aws_lambda_powertools/warnings/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""Shared warnings that don't belong to a single utility"""
+
+
+class PowertoolsWarning(UserWarning):
```

Comment:
```suggestion
class PowertoolsUserWarning(UserWarning):
```

This makes it more aligned with `UserWarning` as that's broadly applied to many types of warning. 

We should clean up the docstring to make it clearer and shorter -- what kind of use cases we raise this warning etc.

Reply:
Done.

Reply:
Done